modin/core/io/text/text_file_dispatcher.py

```diff
@@ -43,6 +43,8 @@
 class TextFileDispatcher(FileDispatcher):
     """Class handles utils for reading text formats files."""
 
+    read_callback = None
```

Comment:
would be awesome if you could add a comment on what this field is expected to do

Reply:
same comment as above